report.md

```diff
@@ -64,6 +61,104 @@ Plan for refactoring complex code:
 
 Estimated impact of refactoring (lower CC, but other drawbacks?).
 
+Method InlineDelegateByteBuddyMockMaker::InlineDelegateByteBuddyMockMaker():
+
+- CCN: 24
+
+- Possible refactor: a large part at the beginning (lines 263 to 265) can be extracted into a separate function. This code deals with initialization error and is independent from the object creation. This part greatly lowers the CC yet has no real drawback. With only this refactor, the function is still above 15 CCN according to lizard's computation method. If needed, we can also refactor the lambda method to assign `isMockConstruction`  (lines 293 to 337) into a separate method.
```

Comment:
The lines seem incorrect. I think the 'large part in the beginning' points to 226-266, and 'isMockConstruction' is on lines 271-292?

Reply:
Yes, I'll update this for the large part at the beginning. The second is correct, but the name isn't well chosen, as there is a variable similar a bit above.

Reply:
It is also possible to refactor the lambda for the isMockConstruction variable, but that would be more than needed I think.